canonical/ubuntu.com
# 23 04 - release prep

## Done

- Preparation for release!

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
  - http://0.0.0.0:8001/

## Issue / Card

Fixes:
- WD-3013
- WD-3012
- WD-3011
- WD-3010
- WD-3009
- WD-3008
- WD-3006
